config/build.ts

```diff
@@ -40,31 +40,50 @@ const defaultApiSets = {
 };
 
 
-(() => {
-    Promise.resolve()
-        .then(processSnippets)
+(async() => {
+    await Promise.resolve()
+        .then(() => processSnippets('samples'))
         .then(updateModifiedFiles)
         .then(checkSnippetsForUniqueIDs)
-        .then(generatePlaylists)
+        .then(() => generatePlaylists('playlists'))
         .then(() => {
             if (accumulatedErrors.length > 0) {
                 throw accumulatedErrors;
             }
         })
         .then(() => {
             banner('Done!', null, chalk.bold.green);
-            process.exit(0);
         })
         .catch(handleError);
+
+    processedSnippets.clear();
```

Comment:
Please change this to a local variable instead of global, other than this change it looks good.